### PR TITLE
docs: add release instructions to README.md

### PR DESCRIPTION
**Pull request by jeffsmale90**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,50 @@
...
+
+Create the package using [npm-pack](https://docs.npmjs.com/cli/v7/commands/npm-pack), which will build the project, and output `trufflesuite-uws-js-unofficial-<version>.tgz`. There will likely be a number of `clang: error:`s ouput, which you can safely ignore.
+
+This file `trufflesuite-uws-js-unofficial-<version>.tgz` is what will be published to npm, and will be served to clients from npm via `npm install`.
+
+### Test the package in Ganache
+
+This can be installed directly to a local nodejs project via `npm install <path-to-trufflesuite-uws-js-unofficial-<version>.tgz>`. In order to test the package within Ganache, navigate to a local clone of Ganache, and install the package directly:
+
+    npm install <path-to-trufflesuite-uws-js-unofficial-[version].tgz>
```
Comment (davidmurdoch) [inline README.md:35]: This part isn't quite accurate, as the module really should be updated in many packages (not in the root) 🤔

**Review by davidmurdoch — CHANGES_REQUESTED** [review]

**Review by cds-amal — COMMENTED** [review]
Drive by review...

This is pretty cool. I was confused by the need to fork from upstream. Perhaps we can  document the context of ganache's build process, which allows for skipping the make step and ignoring the git submodule setup business needed by upstream.

File: README.md (hunk 1 above)
Comment (jeffsmale90) [reply README.md:35]: Good point - I'll clarify this.

**Review by davidmurdoch — APPROVED** [review]

**Review by cds-amal — APPROVED** [review]
Thanks, that looks good to me

**Pull request merged**